test/SolidityExpressionCompiler.cpp

```diff
@@ -72,8 +72,8 @@ class FirstExpressionExtractor: private ASTVisitor
 	Expression* m_expression;
 };
 
-Declaration const& resolveDeclaration(vector<string> const& _namespacedName,
-											  NameAndTypeResolver const& _resolver)
```

Comment:
If if interpreste this correctly in github, then it looks like a lot of white spaces